src/main/java/pl/antonina/tasks/allegro/offer/api/SellingMode.java

```diff
@@ -0,0 +1,17 @@
+package pl.antonina.tasks.allegro.offer.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SellingMode {
```

Comment:
I just started to wonder... Why SellingMode has only one property? Will it have more?

Reply:
SellingMode has few properties (from allegro API), but I need only Price. 
"sellingMode": {
"format": "BUY_NOW",
"price": {},
"fixedPrice": {},
"popularity": 38,
"bidCount": 12
}